lib/mosaic-docker/src/main/java/org/eclipse/mosaic/lib/docker/DockerRun.java

```diff
@@ -99,10 +103,19 @@ public DockerRun user(String user) {
     public DockerRun currentUser() {
         String user = null;
 
-        // Currently, default user is set on Linux, only.
+        // Currently, default user is set on Unix systems, only.
         if (SystemUtils.IS_OS_UNIX) {
-            UnixSystem system = new UnixSystem();
-            user = String.format("%d:%d", system.getUid(), system.getGid());
+            // The class "com.sun.security.auth.module.UnixSystem" is not available on Windows.
+            // Therefore, its import statement has to be bypassed on Windows.
+            try {
+                Class<?> systemClass = Class.forName("com.sun.security.auth.module.UnixSystem");
+                Method getUid = systemClass.getDeclaredMethod("getUid", new Class[]{});
+                Method getGid = systemClass.getDeclaredMethod("getGid", new Class[]{});
+                Object system = systemClass.newInstance();
+                user = String.format("%d:%d", getUid.invoke(system), getGid.invoke(system));
```

Comment:
I'm not sure if that's necessary but I think casting the results of the `invoke()` methods to `Long` makes
it a little bit nicer as `invoke` just returns a `Object`-class:
`user = String.format("%d:%d", (Long) getUid.invoke(system), (Long) getGid.invoke(system));`